src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -133,7 +133,7 @@
                         "PATTERN_VARIABLE_DEF", "RECORD_DEF", "LITERAL_RECORD",
                         "RECORD_COMPONENTS", "RECORD_COMPONENT_DEF", "COMPACT_CTOR_DEF",
                         "TEXT_BLOCK_LITERAL_BEGIN", "TEXT_BLOCK_CONTENT",
-                        "TEXT_BLOCK_LITERAL_END").collect(
+                        "TEXT_BLOCK_LITERAL_END", "LITERAL_YIELD", "SWITCH_RULE").collect(
```

Comment:
place them both to new line.
It make sense to put `.collect(` to new line probably to reduce diff next time.

Reply:
Fixed.